doc/processing_publishing_api_messages.md

```diff
@@ -0,0 +1,39 @@
+# Processing messages from the PublishingAPI
+
+To detect and track changes to content we process messages from the PublishingAPI and update the dimensions editions table. This process is referred to as `Stream` within the codebase. An overview of the process is as follows:
+
+- PublishingAPI publishes messages about content changes to a queue
```

Comment:
It might be useful here to be more specific, and say the `published_documents` exchange. As the Publishing API doesn't publish messages to a queue.